integration/qwen3vl_test.go

```diff
@@ -0,0 +1,266 @@
+//go:build integration
+
+package integration
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ollama/ollama/api"
+)
+
+// TestQwen3VLStreaming tests Qwen3-VL with streaming enabled
+func TestQwen3VLStreaming(t *testing.T) {
```

Comment:
I don't think we want these tests to be specific to qwen3-vl. Instead, I'd think of it as adding capability tests that we can run against _any_ model. @pdevine can help refactor them to be more generic